skt/__init__.py

```diff
@@ -438,22 +438,21 @@ def bisect_iter(self, bad):
 
 
 class KernelBuilder(object):
-    def __init__(self, path, basecfg, cfgtype=None, makeopts=None,
+    def __init__(self, path, basecfg, cfgtype=None, extra_make_args=None,
                  enable_debuginfo=False):
         self.path = path
         self.basecfg = basecfg
         self.cfgtype = cfgtype if cfgtype is not None else "olddefconfig"
         self._ready = 0
-        self.makeopts = None
         self.buildlog = "%s/build.log" % self.path
-        self.defmakeargs = ["make", "-C", self.path]
+        self.make_argv_base = ["make", "-C", self.path]
         self.enable_debuginfo = enable_debuginfo
 
-        if makeopts is not None:
-            # FIXME: Might want something a bit smarter here, something that
-            # would parse it the same way bash does
```

Comment:
This FIXME should be removed in the commit that switches to shlex.

Reply:
I agree. @major can you move that to the other commit please?

Reply:
Will do.